bricksrc/quantities.py

```diff
@@ -1,109 +1,565 @@
+                RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                RDFS.label: Literal("Wind_Direction"),
+            }
+        }
+    },
+    "Energy": {
+        "subconcepts": {
+            "Electric_Energy": {OWL.sameAs: QUDTQK["Energy"]},
+            "Thermal_Energy": {OWL.sameAs: QUDTQK["ThermalEnergy"]},
+        },
+    },
+    "Flow": {
+        OWL.sameAs: QUDTQK["VolumeFlowRate"],
+        "subconcepts": {
+            "Flow_Loss": {
+                QUDT.applicableUnit: [UNIT["M3-PER-SEC"]],
+                QUDT.hasDimensionVector: QUDTDV["A0E0L3I0M0H0T-1D0"],
+                SKOS.definition: Literal(
+                    "The amount of flow rate that is lost during distribution"
+                ),
+                RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                RDFS.label: Literal("FlowLoss"),
+                SKOS.broader: BRICK.Flow,
             },
         },
     },
-    "Daytime": {},
-    "Dewpoint": {},
-    "Direction": {"subclasses": {"Wind_Direction": {}}},
-    "Energy": {"subclasses": {"Electric_Energy": {}, "Thermal_Energy": {}}},
-    "Flow": {"subclasses": {"Flow_Loss": {}}},
-    "Frequency": {"subclasses": {"Alternating_Current_Frequency": {}}},
-    "Humidity": {"subclasses": {"Relative_Humidity": {}}},
-    "Illuminance": {},
-    "Irradiance": {"subclasses": {"Solar_Irradiance": {}}},
+    "Frequency": {
+        OWL.sameAs: QUDTQK["Frequency"],
+        "subconcepts": {"Alternating_Current_Frequency": {}},
+    },
+    "Humidity": {
+        "subconcepts": {
+            "Relative_Humidity": {OWL.sameAs: QUDTQK["RelativeHumidity"]},
+            "Absolute_Humidity": {OWL.sameAs: QUDTQK["AbsoluteHumidity"]},
+        }
+    },
+    "Illuminance": {OWL.sameAs: QUDTQK["Illuminance"]},
+    "Irradiance": {
+        QUDT.applicableUnit: [
+            UNIT["W-PER-M2"],
+            UNIT["W-PER-IN2"],
+            UNIT["W-PER-FT2"],
+            UNIT["W-PER-CeniM2"],
+        ],
+        SKOS.definition: Literal(
+            "The power per unit area of electromagnetic radiation incident on a surface"
+        ),
+        RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+        RDFS.label: Literal("Irradiance"),
+        SKOS.broader: QUDTQK.PowerPerArea,
+        "subconcepts": {
+            "Solar_Irradiance": {
+                QUDT.applicableUnit: [
+                    UNIT["W-PER-M2"],
+                    UNIT["W-PER-IN2"],
+                    UNIT["W-PER-FT2"],
+                    UNIT["W-PER-CeniM2"],
+                ],
+                SKOS.definition: Literal(
+                    "The power per unit area of solar electromagnetic radiation incident on a surface"
+                ),
+                RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                RDFS.label: Literal("SolarIrradiance"),
+                SKOS.broader: BRICK.Irradiance,
+            }
+        },
+    },
     "Level": {
```

Comment:
Do we need this?

Reply:
What part are you referring to? It is ambiguous from the code snippet

Reply:
I meant `Level`.

Reply:
I've moved "Concentration"-oriented quantities to the Concentration part. Level is used for things like water levels, precipitation levels, etc